src/store/modules/job/mutation-types.ts

```diff
@@ -7,4 +7,6 @@ export const JOB_DESCRIPTION_UPDATED = SN_JOB + '/DESCRIPTION_UPDATED'
 export const JOB_HISTORY_UPDATED = SN_JOB + '/HISTORY_UPDATED'
 export const JOB_RUNNING_UPDATED = SN_JOB + '/RUNNING_UPDATED'
 export const JOB_CURRENT_UPDATED = SN_JOB + '/CURRENT_UPDATED'
-export const JOB_MISCELLANEOUS_UPDATED = SN_JOB + '/MISCELLANEOUS_UPDATED'
\ No newline at end of file
+export const JOB_MISCELLANEOUS_UPDATED = SN_JOB + '/MISCELLANEOUS_UPDATED'
+export const JOB_MORE_PENDING_UPDATED = SN_JOB + '/MORE_PENDING_UPDATED'
+export const JOB_MORE_DRAFT_UPDATED = SN_JOB + '/MORE_DRAFT_UPDATED'
```

Comment:
I think we can use a single mutation for updating more jobs.

Reply:
Ok sir